cpp/velox/compute/VeloxBackend.cc

```diff
@@ -62,16 +62,25 @@ using namespace facebook;
 namespace gluten {
 
 namespace {
-MemoryManager* veloxMemoryManagerFactory(std::unique_ptr<AllocationListener> listener) {
-  return new VeloxMemoryManager(std::move(listener));
+MemoryManager* veloxMemoryManagerFactory(const std::string& kind, std::unique_ptr<AllocationListener> listener) {
+  return new VeloxMemoryManager(kind, std::move(listener));
+}
+
+void veloxMemoryManagerReleaser(MemoryManager* memoryManager) {
+  delete memoryManager;
 }
```

Comment:
My original thought is we can use some scope ptr for this, since memory manager is `static`

Reply:
> My original thought is we can use some scope ptr for this, since memory manager is `static`

Auto ptr management is not possible for this part of code at least at the moment, since the lifecycle is managed over JNI bridge.

What we can do is to centralize the `new` and `delete` operations to one place as a better design.